src/page/shipping-step.page.ts

```diff
@@ -5,8 +5,14 @@ export class ShippingStepPage {
   private proceedCheckoutButton: ElementFinder;
 
   constructor () {
-    this.acceptTermsOfServiceCheckbox = $('#cgv');
-    this.proceedCheckoutButton = $('#form > p > button > span');
+    /* I think the selector of the checkbox could be left like it is, but the name is not
+    really clear and maybe can be confused if for some other reasons other checkbocks are inserted.
+    So, I add the specification of where it is supposed to be located */
+    this.acceptTermsOfServiceCheckbox = $('p.checkbox > #uniform-cgv #cgv');
+
+    /* Respecting this button, it has te same problem as the one on the addres step page,
+    so I will correct it the same way. */
+    this.proceedCheckoutButton = $('#form > p.cart_navigation > button[name="processCarrier"] > span');
```

Comment:
It's true. I thought maybe it was a good idea to leave at least te p.cart_navgitation since another button with the same name could appear at some other place (Like happens on the summary step page), but in the end I decided to leave just the button[name="processCarrier"]. What is the best practice in that case?